darwin.c

```diff
@@ -7,82 +7,8 @@
 #include <mach-o/dyld.h>
 #include "common.h"
 
-int elevate(char *path, char *prompt, char *iconPath)
```

Comment:
Yeah, I don't see that as such a big downside since we have an existing scheme (byteexec) for dealing with that.